website/content/docs/plugins/install-plugins.mdx

```diff
@@ -144,9 +148,34 @@ The source address with all three components given explicitly is called the
 plugin's _fully-qualified address_. You will see fully-qualified address in
 various outputs, like error messages.
 
-## Plugin location
+## Plugin Installation Workflow
 
-@include "plugins/plugin-location.mdx"
+* [`packer init`](/packer/docs/commands/init) will install plugins in the **last** directory
+in the following numbered list.
+
+1. `PACKER_PLUGINS_PATH` if set will be the sole location for installing plugins. All other 
+plugin directories will be ingored. 
+1. `PACKER_CONFIG_DIR`\plugins on Windows systems, or `PACKER_CONFIG_DIR`/plugins on all other systems.
+
+* During the initialization of Packer, any plugin required in the
+**`required_plugins`** section will be looked up in all entries of the following
+list. **First** plugin found takes precedence. Two binaries of the same plugin
+with two different version will be considered as two different plugins. Highest
+found version matching `required_plugins` will be taken into consideration.
+
+During initialization, on a `darwin_amd64` system, Packer will look-up for the
+following files:
+
+* `PACKER_PLUGIN_PATH/github.com/azr/happycloud/packer-plugin-happycloud_*_x5.0_darwin_amd64`
+* `PACKER_HOME_DIR/plugins/github.com/azr/happycloud/packer-plugin-happycloud_*_x5.0_darwin_amd64`
+
+The first plugin-name/version files found will take precedence.
+
+For plugins located under the `github.com/azr/happycloud/` directory structure an accompanying SHA256SUM file
+will be required in order for `packer init` to ensure the plugin being loaded has not been tampered with.
+The SHA256SUM file will be automatically generated when a plugin is installed via `packer init` if the plugin
+was installed manually into `PACKER_HOME_DIR/plugins/github.com/azr/happycloud/` then the file
```

Comment:
Shouldn't that be `PACKER_CONFIG_DIR` rather than `PACKER_HOME_DIR` here? The latter is not mentioned elsewhere in the page so I fear it may be confusing if that's the intended variable in this case